integration_test/ops_agent_test/main_test.go

```diff
@@ -96,17 +95,39 @@ func workDirForImage(imageSpec string) string {
 }
 
 func startCommandForImage(imageSpec string) string {
-	if gce.IsWindows(imageSpec) {
+	isUAPPlugin := agents.IsOpsAgentUAPPlugin()
+
+	if gce.IsWindows(imageSpec) && isUAPPlugin {
+		return ""
+	}
+
+	if gce.IsWindows(imageSpec) && !isUAPPlugin {
 		return "Start-Service google-cloud-ops-agent"
 	}
+
+	if !gce.IsWindows(imageSpec) && isUAPPlugin {
+		return "grpcurl -plaintext -d '{}' localhost:1234 plugin_comm.GuestAgentPlugin/Start"
+	}
+
 	// Return a command that works for both < 2.0.0 and >= 2.0.0 agents.
 	return "sudo service google-cloud-ops-agent start || sudo systemctl start google-cloud-ops-agent"
 }
 
 func stopCommandForImage(imageSpec string) string {
-	if gce.IsWindows(imageSpec) {
+	isUAPPlugin := agents.IsOpsAgentUAPPlugin()
```

Comment:
I think we can improve the if structure here by nesting UAP cases together.
if UAP {
...
}